packages/components/src/components/hint-box/hint-box.tsx

```diff
@@ -20,17 +33,28 @@ const HintBox = ({ className, icon, is_info, is_warn, is_danger, message }: THin
                     'dc-hint-box--info': is_info,
                     'dc-hint-box--warn': is_warn,
                     'dc-hint-box--danger': is_danger,
+                    'dc-hint-box--inline': is_inline,
                 },
                 className
             )}
         >
             <Icon
-                className='dc-hint-box__icon'
+                className={classNames('dc-hint-box__icon', {
+                    'dc-hint-box--inline-icon': is_inline,
+                })}
                 custom_color={is_info ? 'var(--status-info)' : undefined}
                 icon={icon}
-                size={16}
+                height={icon_height}
+                width={icon_width}
+                size={icon_height && icon_width ? 0 : 16}
```

Comment:
Can u explain this condition. If icon_height & icon_width are provided, set the size value to 0?

Reply:
Yes if the height and width of the icon is provided then the size should set to 0